ui/qtgalacticplot.py

```diff
@@ -148,5 +159,5 @@ def __update_annotation(self, ind) -> None:
         '''Updates annotation parameters'''
         pos = self.__planetsScatter.get_offsets()[ind["ind"][0]]
         self.__annotate.xy = pos
-        text = "{}".format("\n".join([self.__planetNames[n] for n in ind["ind"]]))
+        text = "".join("\n{} [{}]".format(self.__planetNames[n], self.__planetOwners[n]) for n in ind["ind"])
```

Comment:
Not sure why this creates an extra newline, but it seems to in the screenshot

Reply:
I didn't spot that! Too excited that the faction was showing up where I wanted it.  I wonder if it's because the `\n` is now within the join so it gets iterated an extra time? Weird.

Reply:
Yep, that was it. The `\n` needs to be immediately before the join, apparently. https://note.nkmk.me/en/python-string-line-break/#concatenate-a-list-of-strings-on-new-lines